framework/source/class/qx/ui/splitpane/Pane.js

```diff
@@ -102,6 +102,17 @@ qx.Class.define("qx.ui.splitpane.Pane",
       init  : "horizontal",
       check : [ "horizontal", "vertical" ],
       apply : "_applyOrientation"
+    },
+
+    /**
+     * The visibility of the splitter.
+     * Allows to remove the splitter in favor of other visual separation means like background color differences.
+     */
+    splitterVisible :
```

Comment:
how about calling it

`impliedSplitterMode`

then the logic of the code might be a bit easier to follow.

> **implied**
> /ɪmˈplʌɪd/
> adjective
> suggested but not directly expressed; implicit.
> "she was aware of his implied criticism"

Reply:
Hm - I would find that more difficult to understand than simply "whether the splitter is visible" - but maybe that's just me 😄

Reply:
I would agree with @cboulanger

Reply:
I think is more important to have an accurate, self-describing name... plus that way it allows to store a simple ``Boolean``